src/screens/wallet/token-list/token-list.tsx

```diff
@@ -72,6 +79,18 @@ export const TokenList: FC = () => {
     [isShowTezos, filteredAssetsList]
   );
 
+  const screenFillingItemsCount = flatlistHeight / ITEM_HEIGHT;
+
+  const renderData =
+    isAndroid && screenFillingItemsCount > flatListData.length
+      ? flatListData.concat(Array(Math.ceil(screenFillingItemsCount - flatListData.length)).fill(emptyToken))
+      : flatListData;
+
+  const handleLayout = (event: LayoutChangeEvent) => {
+    const { height } = event.nativeEvent.layout;
```

Comment:
why this destructuring is needed?  `height` variable is used in sigle place 
```
const handleLayout = (event: LayoutChangeEvent) => setFlatlistHeight(event.nativeEvent.layout.height);
```
this code looks clearer